src/lib/signature_lib/checked.ml

```diff
@@ -97,7 +111,10 @@ module type S = sig
       -> (unit, _) checked
   end
 
+  val get_x : curve -> bool list
```

Comment:
Better to not have these functions and just get x, y using `to_affine_coordinates` and then use `Field.to_bits`.